From a252330d53afad6f8a4645933989bb017dc35ad8 Mon Sep 17 00:00:00 2001
From: Victor Stinner <victor.stinner@gmail.com>
Date: Thu, 27 Apr 2017 16:08:24 +0200
Subject: [PATCH] bpo-30175: Skip client cert tests of test_imaplib

The IMAP server cyrus.andrew.cmu.edu doesn't accept our randomly
generated client x509 certificate anymore.
---
 Lib/test/test_imaplib.py | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/Lib/test/test_imaplib.py b/Lib/test/test_imaplib.py
index 5ef9512c3f5..a33fb714fe7 100644
--- a/Lib/test/test_imaplib.py
+++ b/Lib/test/test_imaplib.py
@@ -965,6 +965,9 @@ def test_logincapa(self):
             _server = self.imap_class(self.host, self.port)
             self.check_logincapa(_server)
 
+    @unittest.skipIf(True,
+                     "bpo-30175: FIXME: cyrus.andrew.cmu.edu doesn't accept "
+                     "our randomly generated client x509 certificate anymore")
     def test_logincapa_with_client_certfile(self):
         with transient_internet(self.host):
             with support.check_warnings(('', DeprecationWarning)):
@@ -972,6 +975,9 @@ def test_logincapa_with_client_certfile(self):
                                           certfile=CERTFILE)
                 self.check_logincapa(_server)
 
+    @unittest.skipIf(True,
+                     "bpo-30175: FIXME: cyrus.andrew.cmu.edu doesn't accept "
+                     "our randomly generated client x509 certificate anymore")
     def test_logincapa_with_client_ssl_context(self):
         with transient_internet(self.host):
             _server = self.imap_class(
